File: sample-videochat-webrtc/sample-videochat-webrtc/Components/SharingViewController/Views/SharingCell.m
//
//  SharingCell.m
//  sample-videochat-webrtc
//
//  Created by Injoit on 2/25/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

#import "SharingCell.h"

@interface SharingCell()
//MARK: - IBOutlets
@property (weak, nonatomic) IBOutlet UIImageView *imagePreview;
@end

@implementation SharingCell
//MARK - Setup
- (void)setupImageName:(NSString *)imageName {
    self.imagePreview.image = [UIImage imageNamed:imageName];
}

@end

File: sample-videochat-webrtc-swift/sample-videochat-webrtc-swift/Components/SharingViewController/SharingCell.swift
//
//  SharingCell.swift
//  sample-videochat-webrtc-swift
//
//  Created by Injoit on 28.12.2020.
//  Copyright © 2020 QuickBlox. All rights reserved.
//

import UIKit

class SharingCell: UICollectionViewCell {

    @IBOutlet private weak var imagePreview: UIImageView!
    
    var imageName = "" {
        didSet {
            imagePreview.image = UIImage(named: imageName)
        }
    }
}
